staging/src/k8s.io/kubectl/pkg/cmd/drain/drain.go

```diff
@@ -321,12 +319,21 @@ func (o *DrainCmdOptions) RunDrain() error {
 
 func (o *DrainCmdOptions) deleteOrEvictPodsSimple(nodeInfo *resource.Info) error {
 	list, errs := o.drainer.GetPodsForDeletion(nodeInfo.Name)
+	if len(list.Pods()) > 0 && o.drainer.DryRun {
+		fmt.Fprintf(o.Out, "namespace/pod (dry run)\n")
+		for _, pod := range list.Pods() {
+			fmt.Fprintf(o.Out, "%s/%s\n", pod.Namespace, pod.Name)
```

Comment:
From what I see `DeleteOrEvicPods` is doing this: https://github.com/kubernetes/kubernetes/blob/ab0bfb304caa3f67d5c207c2356557cda90936b9/staging/src/k8s.io/kubectl/pkg/drain/drain.go#L211 so I'd expect something similar for printing here maybe with `(dry run)` added at the end.

Reply:
Although we should probably update that to print both namespace and name altogether, in which case what you have here would be in sync.

Reply:
Honestly, in both cases, we should probably fix that helper method to be reusable by passing `ResourcePrinter` but let's leave that as a separate PR :wink:

Reply:
I added namespace to kubectl/pkg/drain/drain.go added "evicting pod ... (dry-run)"  to dry-run list. 

```console
$ kubectl drain ip-10-0-144-193.us-east-2.compute.internal --dry-run --pod-selector="app=hello-node"
node/ip-10-0-144-193.us-east-2.compute.internal cordoned (dry run)
evicting pod "test"/"hello-node-7676b5fb8d-6w57r" (dry run)
node/ip-10-0-144-193.us-east-2.compute.internal drained (dry run)
```
non-dry-run:
```console
$ kubectl drain ip-10-0-144-193.us-east-2.compute.internal --pod-selector="app=hello-node"
node/ip-10-0-144-193.us-east-2.compute.internal cordoned
evicting pod "test"/"hello-node-7676b5fb8d-6w57r"
pod/hello-node-7676b5fb8d-6w57r evicted
node/ip-10-0-144-193.us-east-2.compute.internal evicted
```

Reply:
but will change to %q if that's preferred